base/kra/src/main/java/com/netscape/cms/servlet/csadmin/GetTransportCert.java

```diff
@@ -133,16 +135,20 @@ protected void process(CMSRequest cmsReq) throws EBaseException {
         // send success status back to the requestor
         try {
             logger.debug("GetTransportCert: Sending response " + mime64);
-            XMLObject xmlObj = new XMLObject();
-            Node root = xmlObj.createRoot("XMLResponse");
 
-            xmlObj.addItemToContainer(root, "Status", SUCCESS);
-            xmlObj.addItemToContainer(root, "TransportCert", mime64);
-            byte[] cb = xmlObj.toByteArray();
+            ObjectMapper mapper = new ObjectMapper()
+                    .enable(SerializationFeature.INDENT_OUTPUT)
+                    .setAnnotationIntrospector(new JacksonAnnotationIntrospector());
```

Comment:
I don't suppose this is necessary if we're about to convert to `byte[]` and fire it off, without printing the `JSON` anywhere.